README
NAME
    Pod::Weaver::Section::BugsAndLimitations - Add a BUGS AND LIMITATIONS
    pod section

VERSION
    version 1.102670

SYNOPSIS
    In "weaver.ini":

        [BugsAndLimitations]

OVERVIEW
    This section plugin will produce a hunk of Pod that refers to the
    bugtracker URL.

    You need to use Dist::Zilla::Plugin::Bugtracker in your "dist.ini" file,
    because this plugin relies on information that other plugin generates.

FUNCTIONS
  weave_section
    adds the "BUGS AND LIMITATIONS" section.

INSTALLATION
    See perlmodinstall for information and options on installing Perl
    modules.

BUGS AND LIMITATIONS
    No bugs have been reported.

    Please report any bugs or feature requests through the web interface at
    <http://rt.cpan.org>.

AVAILABILITY
    The latest version of this module is available from the Comprehensive
    Perl Archive Network (CPAN). Visit <http://www.perl.com/CPAN/> to find a
    CPAN site near you, or see
    <http://search.cpan.org/dist/Pod-Weaver-Section-BugsAndLimitations/>.

    The development version lives at
    <http://github.com/hanekomu/Pod-Weaver-Section-BugsAndLimitations> and
    may be cloned from
    <git://github.com/hanekomu/Pod-Weaver-Section-BugsAndLimitations>.
    Instead of sending patches, please fork this project using the standard
    git and github infrastructure.

AUTHOR
    Marcel Gruenauer <marcel@cpan.org>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2010 by Marcel Gruenauer.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.



Changes
Revision history for Perl extension Pod-Weaver-Section-BugsAndLimitations

1.102670  2010-09-24 17:22:26 Europe/Vienna
    - use the correct hash key to access bug tracker URL (Mike Doherty)

1.102460  2010-09-03 12:26:40 Europe/Vienna
   - separated from Pod-Weaver-PluginBundle-MARCEL 


lib/Pod/Weaver/Section/BugsAndLimitations.pm
use 5.008;
use strict;
use warnings;

package Pod::Weaver::Section::BugsAndLimitations;
BEGIN {
  $Pod::Weaver::Section::BugsAndLimitations::VERSION = '1.102670';
}

# ABSTRACT: Add a BUGS AND LIMITATIONS pod section
use Moose;
with 'Pod::Weaver::Role::Section';
use namespace::autoclean;
use Moose::Autobox;

sub weave_section {
    my ($self, $document, $input) = @_;
    my $bugtracker = $input->{zilla}->distmeta->{resources}{bugtracker}{web}
      || 'http://rt.cpan.org';
    $document->children->push(
        Pod::Elemental::Element::Nested->new(
            {   command  => 'head1',
                content  => 'BUGS AND LIMITATIONS',
                children => [
                    Pod::Elemental::Element::Pod5::Ordinary->new(
                        {   content => <<EOPOD,
No bugs have been reported.

Please report any bugs or feature requests through the web interface at
L<$bugtracker>.
EOPOD
                        }
                    ),
                ],
            }
        ),
    );
}
1;


__END__
=pod

=for test_synopsis 1;
__END__

=head1 NAME

Pod::Weaver::Section::BugsAndLimitations - Add a BUGS AND LIMITATIONS pod section

=head1 VERSION

version 1.102670

=head1 SYNOPSIS

In C<weaver.ini>:

    [BugsAndLimitations]

=head1 OVERVIEW

This section plugin will produce a hunk of Pod that refers to the bugtracker
URL.

You need to use L<Dist::Zilla::Plugin::Bugtracker> in your C<dist.ini> file,
because this plugin relies on information that other plugin generates.

=head1 FUNCTIONS

=head2 weave_section

adds the C<BUGS AND LIMITATIONS> section.

=head1 INSTALLATION

See perlmodinstall for information and options on installing Perl modules.

=head1 BUGS AND LIMITATIONS

No bugs have been reported.

Please report any bugs or feature requests through the web interface at
L<http://rt.cpan.org>.

=head1 AVAILABILITY

The latest version of this module is available from the Comprehensive Perl
Archive Network (CPAN). Visit L<http://www.perl.com/CPAN/> to find a CPAN
site near you, or see L<http://search.cpan.org/dist/Pod-Weaver-Section-BugsAndLimitations/>.

The development version lives at L<http://github.com/hanekomu/Pod-Weaver-Section-BugsAndLimitations>
and may be cloned from L<git://github.com/hanekomu/Pod-Weaver-Section-BugsAndLimitations>.
Instead of sending patches, please fork this project using the standard
git and github infrastructure.

=head1 AUTHOR

Marcel Gruenauer <marcel@cpan.org>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2010 by Marcel Gruenauer.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=cut



Makefile.PL

use strict;
use warnings;

BEGIN { require 5.008; }

use ExtUtils::MakeMaker 6.31;



my %WriteMakefileArgs = (
  'ABSTRACT' => 'Add a BUGS AND LIMITATIONS pod section',
  'AUTHOR' => 'Marcel Gruenauer <marcel@cpan.org>',
  'BUILD_REQUIRES' => {
    'Carp' => '0',
    'English' => '0',
    'File::Find' => '0',
    'File::Temp' => '0',
    'Scalar::Util' => '0',
    'Test::More' => '0.94'
  },
  'CONFIGURE_REQUIRES' => {
    'ExtUtils::MakeMaker' => '6.31'
  },
  'DISTNAME' => 'Pod-Weaver-Section-BugsAndLimitations',
  'EXE_FILES' => [],
  'LICENSE' => 'perl',
  'NAME' => 'Pod::Weaver::Section::BugsAndLimitations',
  'PREREQ_PM' => {
    'Dist::Zilla::Plugin::Bugtracker' => '0',
    'Moose' => '0',
    'Moose::Autobox' => '0',
    'Pod::Weaver::Role::Section' => '0',
    'namespace::autoclean' => '0'
  },
  'VERSION' => '1.102670',
  'test' => {
    'TESTS' => 't/*.t'
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



